okhttp/src/main/java/okhttp3/RealCall.java

```diff
@@ -65,6 +66,16 @@ protected RealCall(OkHttpClient client, Request originalRequest) {
     }
   }
 
+  private void captureCallStackTrace() {
+    StackTraceElement[] stackTrace = Thread.currentThread().getStackTrace();
```

Comment:
Oh, gotcha. Are there any good ways of getting that information? I can also do some research on this.